package.json

```diff
@@ -53,7 +53,7 @@
     "universal-cookie": "4.0.4"
   },
   "devDependencies": {
-    "@edx/frontend-build": "12.0.1",
+    "@edx/frontend-build": "11.0.2",
```

Comment:
Reverting this back to latest v11 to avoid using 12.0.1 which turns out doesn't work as intended and 12.0.2 introduces a couple hundred new ESLint errors (most can be auto-fixed).